docs/examples/day-night-masking.py

```diff
@@ -0,0 +1,197 @@
+"""
+Day-Night Masking
+=================
+
+Masking day-night periods using the PVAnalytics daytime module.
+"""
+
+# %%
+# Identifying and masking day-night periods in an AC power time series or
+# irradiance time series can aid in future data analysis, such as detecting
+# if a time series has daylight savings time or time shifts.
+
+import pvanalytics
+from pvanalytics.features.daytime import power_or_irradiance
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+import pvlib
+import numpy as np
+
+# %%
+# First, read in the 1-minute sampled AC power time series data, taken
+# from the SERF East installation on the NREL campus.
+# This sample is provided from the NREL PVDAQ database, and contains
+# a column representing an AC power data stream.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+ac_power_file = pvanalytics_dir / 'data' / 'serf_east_1min_ac_power.csv'
+data = pd.read_csv(ac_power_file, index_col=0, parse_dates=True)
+data = data.sort_index()
+# This is the known frequency of the time series. You may need to infer
+# the frequency or set the frequency with your AC power time series.
+freq = "1T"
+# These are the latitude-longitude coordinates associated with the
+# SERF East system.
+latitude = 39.742
+longitude = -105.173
+# Plot the time series.
+data['ac_power__752'].plot()
+plt.xticks(rotation=20)
+plt.xlabel("Date")
+plt.ylabel("AC Power (kW)")
+plt.tight_layout()
+plt.show()
+
+# %%
+# The pvlib.solarposition.sun_rise_set_transit_spa function is
```

Comment:
Could use `:py:func:` here to link out to the pvlib docs

Reply:
Updated in commit https://github.com/pvlib/pvanalytics/pull/139/commits/8fca43cd2428c7d279072138e79cdd7043e47b62